docs/en/core-services/Catalog.md

```diff
@@ -102,17 +102,28 @@ $ ${ALLUXIO_HOME}/bin/alluxio table attachdb --db alluxio_db hive \
     thrift://metastore_host:9083 default
 ```
 
-If you want specify config file for the UDB, please add `attachdb` option `-o catalog.db.config.file`.
-Each time you want to affect the config file, you can use `alluxio table sync`.
-
-If you want to write a configuration file for the UDB, please configure it like the following given example.
-It must contain an array `bypassSet`, the comma-separated table names will be bypassed.
-
-```json
-{
-  "bypassSet" : ["table1", "table2"]
-}
-```
+If you want to specify a config file for the UDB, please append an option `-o catalog.db.config.file` to
+`attachdb` command.
+After each time you made changes to the config file, you can use `alluxio table sync` to apply the changes.
+
+The configuration file is in JSON format, and can contain these configurations:
+
+1. Tables and partitions bypassing specification:
```

Comment:
please explain here what do we mean by "Tables and partitions bypassing "

Reply:
Added in 9ad0139.